types/hapi__joi/hapi__joi-tests.ts

```diff
@@ -1158,6 +1158,13 @@ schema = Joi.symbol().map({
 
 // --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- ---
 
+const rule = Joi.string().case('upper').$_getRule('case');
+if (rule && rule.args) {
+    rule.args.direction;
```

Comment:
Is this supposed to be return?

Reply:
Yeah, if it would be preferable to capture in a variable like the following then I can update with that.
```typescript 
const direction = rule.args.direction;
```

Reply:
Sure, sounds good